fastapi__template/database.py

```diff
@@ -19,7 +19,14 @@
 
 def get_engine():
     """Get database engine."""
-    return create_engine(SETTINGS.database_url)
```

Comment:
This could be improved to a Dict mapping, and if the engine mapping returns none then an error could be raised.

eg:

DATABASE_ENGINES = {
"postgres": create_engine(...)
...
}